tools/c2chapel/c2chapel.py

```diff
@@ -136,6 +142,9 @@ def getArgs():
                         action="store_true")
     parser.add_argument("-V", "--version", action="version", version="%(prog)s " + __version__)
     parser.add_argument("cppFlags", nargs="*", help="flags forwarded to the C preprocessor (invoked with cc -E)")
+    parser.add_argument("--use-gnu",
+                        help="use gnu parser instead of pycparser with c2chapel",
+                        action="store_true")
```

Comment:
I'm not sure that `--use-gnu` is very intuitive to me. At first read I interpret it to mean parse with the gnu compiler or something. I wonder if something like `--allow-gnu-extensions` or just `--gnu-extensions` would be more obvious?

I'm also not sure how much the fact that we use pycparser is being used is exposed to users so we might want to make the help message more generic. Also note that the `file` arg help says `C99 file` and this flag will enable some gnu extensions so we may need to update the wording there a little too.

Reply:
Alright, I think `--allow-gnu-extensions` makes sense, I'll make that change. For the `C99 file` part, what additional info do you think should be added? I am not very familiar with how these C99/GNU/other C versions relate, so I'm struggling to come up with an intelligent new helper message. Something simple like `C99 or GNU file` wouldn't make sense, since GNU isn't just one standard like that, right?

Reply:
I couldn't think of a good file message either. Let's just leave that alone for now and just update the `--gnu-extensions` one. That should make it obvious enough that it's C99 by default, but this allows some gnu extensions

Reply:
I believe https://github.com/chapel-lang/chapel/blob/main/doc/rst/tools/c2chapel/c2chapel.rst will have to be updated too BTW.